src/handbook/customer/sales/subscription-agreement-1.5.njk

```diff
@@ -0,0 +1,526 @@
+---
+# Note: Do not delete this file. It's still referenced in contracts and quotes.
```

Comment:
```suggestion
```

Reply:
@Yndira-E We need to keep this line, as the URL will be used to link to the contract. As such we cannot change it, nor remove it.